src/components/pages/roster-page/roster-page.tsx

```diff
@@ -78,7 +80,10 @@ export const RosterPage = () => {
   const [rowsPerPage, setRowsPerPage] = useState(10);
   const [unitNameMap, setUnitNameMap] = useState<{[key: string]: string}>({});
   const [selectedRosterEntry, setSelectedRosterEntry] = useState<ApiRosterEntry>();
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [alertDialogProps, setAlertDialogProps] = useState<AlertDialogProps>({ open: false });
```

Comment:
It looks like the success alerts are gone, which is why `alertDialogProps` was unused. I guess they probably got stripped out with the error dialog and you forgot to add them back in.

It should never be necessary to disable that eslint rule on a piece of React state like that, since an unused variable there is a sign that something is broken or unnecessary.

Reply:
Yikes! That's not good... I'll make sure those are fixed and take another pass through the changes to see if any similar problems are lurking.

Reply:
I resolved this by adding the `<AlertDialog>` back to the page. I created a new `<ErrorDialog>` component for the `<ErrorBoundary>` so success and error dialog components can co-exist.